textlint/index.js

```diff
@@ -0,0 +1,186 @@
+				'Youtube',
+				[ 'Common[ .]js', 'CommonJS' ],
+				[ 'JSDocs?', 'JSDoc' ],
+				[ 'Node(?:js)?', 'Node.js' ],
+				[ 'React[ .]js', 'React' ],
+				[ 'SauceLabs', 'Sauce Labs' ],
+				[ 'StackOverflow', 'Stack Overflow' ],
+				[ 'styled ?components', 'styled-components' ],
+				[ 'HTTP[ /]2(?:\\.0)?', 'HTTP/2' ],
+				[ 'OS X', 'macOS' ],
+				[ 'Mac ?OS', 'macOS' ],
+				[ 'a npm', 'an npm' ],
+				'3D',
+				[ '3-D', '3D' ],
+				'Ajax',
+				'API',
+				[ 'API[\'’]?s', 'APIs' ],
+				'CSS',
+				'GIF',
+				'HTML',
+				'HTTPS',
+				'IoT',
+				'I/O',
+				[ 'I-O', 'I/O' ],
+				'JPEG',
+				'MIME',
+				'OK',
+				'PaaS',
+				'PDF',
+				'PNG',
+				'SaaS',
+				'URL',
+				[ 'URL[\'’]?s', 'URLs' ],
+				[ 'an URL', 'a URL' ],
+				[ 'wi[- ]?fi', 'Wi-Fi' ],
+				'ID',
+				[ 'id[\'’]?s', 'IDs' ],
+				[ 'backwards compatible', 'backward compatible' ],
+				[ 'she or he', 'they' ],
+				[ 'she/he', 'they' ],
+				[ 'he or she', 'they' ],
+				[ 'he/she', 'they' ],
+				[ 'his or hers', 'theirs' ],
+				[ 'his or her', 'their' ],
+				[ 'him or her', 'them' ],
+				[ 'him or herself', 'themself' ],
+				[ '\\(s\\)he', 'they' ],
+				[ 'repo\\b', 'repository' ],
+				[ 'auto[- ]complete', 'autocomplete' ],
+				[ 'back[- ]end(\\w*)', 'backend$1' ],
+				[ 'bug[- ]fix(es)?', 'bugfix$1' ],
+				[ 'change[- ]log(s)?', 'changelog$1' ],
+				[ 'check[- ]box(es)?', 'checkbox$1' ],
+				[ 'code[- ]base(es)?', 'codebase$1' ],
+				[ 'co[- ]locate(d?)', 'colocate$1' ],
+				[ 'end[- ]point(s)?', 'endpoint$1' ],
+				[ 'e[- ]mail(s)?', 'email$1' ],
+				[ 'file[- ]name(s)?', 'filename$1' ],
+				[ 'front[- ]end(\\w*)', 'frontend$1' ],
+				[ 'hack[- ]a[- ]thon(s)?', 'hackathon$1' ],
+				[ 'host[- ]name(s)?', 'hostname$1' ],
+				[ 'hot[- ]key(s)?', 'hotkey$1' ],
+				[ 'life[- ]cycle', 'lifecycle' ],
+				[ 'life[- ]stream(s)?', 'lifestream$1' ],
+				[ 'lock[- ]file(s)?', 'lockfile$1' ],
+				[ 'mark-up', 'markup' ],
+				[ 'meta[- ]data', 'metadata' ],
+				[ 'micro[- ]service(s)?', 'microservice$1' ],
+				[ 'name[- ]space(s)?', 'namespace$1' ],
+				[ 'pre[- ]condition(s)?', 'precondition$1' ],
+				[ 'pre[- ]defined', 'predefined' ],
+				[ 'pre[- ]release(s)?', 'prerelease$1' ],
+				[ 're[- ]write', 'rewrite' ],
+				[ 'run[- ]time', 'runtime' ],
+				[ 'screen[- ]shot(s)?', 'screenshot$1' ],
+				[ 'screen[- ]?snap(s)?', 'screenshot$1' ],
+				[ 'sub[- ]class((?:es|ing)?)', 'subclass$1' ],
+				[ 'sub[- ]tree(s)?', 'subtree$1' ],
+				[ 'time[- ]stamp(s)?', 'timestamp$1' ],
+				[ 'touch[- ]screen(s)?', 'touchscreen$1' ],
+				[ 'user[- ]name(s)?', 'username$1' ],
+				[ 'walk[- ]through', 'walkthrough' ],
+				[ 'white[- ]space', 'whitespace' ],
+				[ 'wild[- ]card(s)?', 'wildcard$1' ],
+				[ 'file-?type(s)?', 'file type$1' ],
+				[ 'micro-?frontend(s)?', 'micro frontend$1' ],
+				[ 'open-?source(ed?)', 'open source$1' ],
+				[ 'regexp?(s)?', 'regular expression$1' ],
+				[ 'style[- ]guide(s)?', 'styleguide$1' ],
```

Comment:
As mentioned in this repo's readme: "styleguide" is wrong be all accounts except mine. It's its own thing, and it should get its own word.